x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -715,11 +715,14 @@ export class TaskRunner<
 
       const { errors: errorsFromLastRun } = this.ruleResult.getLastRunResults();
       if (errorsFromLastRun.length > 0) {
+        const errorMessage = `Executing Rule ${
+          this.ruleType.id
+        }:${ruleId} has resulted in the following error(s): ${errorsFromLastRun.join(',')}`;
+        this.logger.error(errorMessage, {
```

Comment:
I think we sometimes log things we see out of the executors as warnings - we don't really know how bad they are.  Is this such a case?  Or maybe because the executor has populated error and not warning, it's ok to use the error logger vs warn logger.